### PR TITLE
Edit gateway antenna placement in the Console

### LINKED ISSUE #4324: Edit gateway antenna placement
#### Summary

Edit gateway antenna placement

#### Why do we need this?

To specify whether the gateway antenna is indoor or outdoor (or unknown)

#### What is already there? What do you see now?

Editing the gateway location and gain

#### What is missing? What do you want to see?

Editing the antenna placement. This is new since #4323 

#### How do you propose to implement this?

Dropdown where you can currently set the gateway location.

The dropdown is basically Unknown, Indoor or Outdoor. See https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.14/api/gateway.proto#L294-L298

#### How do you propose to test this?

Local testing should suffice

#### Can you do this yourself and submit a Pull Request?

No, please triage

### PR DESCRIPTION
**Pull request by bafonins**
#### Summary

Closes https://github.com/TheThingsNetwork/lorawan-stack/issues/4324

<details>
<summary>Screenshots</summary>
Screenshot 2021-07-27 at 15 22 25

</details>

#### Changes

- Add `<Radio.Group />` for choosing gateway antenna placement

#### Testing

Manual

#### Notes for Reviewers

@johanstokking Please see screenshots

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Review by kschiffer — APPROVED** [review]
Please add the following tooltip:

```
What is this?
Informs whether the gateway antenna is placed inside or outside. This information is used solely for display purposes, e.g. on public gateway maps (if your gateway is set as public) and has no technical or functional effects.
```

@neoaggelos please confirm.

File: pkg/webui/console/containers/gateway-location-form/gateway-location-form.js [hunk 1]
```diff
@@ -65,6 +69,13 @@ const validationSchema = Yup.object().shape({
   }),
   location_public: Yup.bool(),
   update_location_from_status: Yup.bool(),
+  _placement: Yup.string()
+    .oneOf(['PLACEMENT_UNKNOWN', 'INDOOR', 'OUTDOOR'])
+    .when('update_location_from_status', {
```
Comment (johanstokking) [inline pkg/webui/console/containers/gateway-location-form/gateway-location-form.js]: This is unrelated; the location can be updated from status messages, and still the placement can be indoor or outdoor. The status message does not include the placement, only the location.

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: pkg/webui/console/containers/gateway-location-form/gateway-location-form.js (hunk 1 above)
Comment (bafonins) [reply pkg/webui/console/containers/gateway-location-form/gateway-location-form.js]: According to https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.14/api/gateway.proto#L315 `placement` is a part of gateway antennas list. Should we then register a dummy antenna with specific placement if `update_location_from_status` is selected?

Current implementation strips the `antennas` entry if `update_location_from_status` is selected by the user.

Comment (johanstokking) [reply pkg/webui/console/containers/gateway-location-form/gateway-location-form.js]: Yes indeed, we should. It's not a dummy antenna. Antenna has valuable information like placement and gain: it's not only about location (anymore). We can assume that each gateway has at least 1 antenna. If there is no information about antennas, we allow no antennas to be set.

Comment (bafonins) [reply pkg/webui/console/containers/gateway-location-form/gateway-location-form.js]: So, if I understood that correctly setting placement should work as follows:
1. The user sets antenna location manually (setting latitude, longitude, altitude), then we simply add the `placement` field within that antenna.
2. The user selects `update_location_from_status`, then location fields are disabled (latitude, longitude, altitude). If there is an antenna, then we allow setting `placement`, otherwise it is disabled.

Is this correct?

Comment (johanstokking) [reply pkg/webui/console/containers/gateway-location-form/gateway-location-form.js]: Yes. The `update_location_from_status` is only for location.

The user can always configure `gain` and `placement` for the antenna.

**Review by kschiffer — COMMENTED** [review]
The tooltip is still missing, otherwise LGTM.

**Review by johanstokking — APPROVED** [review]
Screenshots and logic LGTM, didn't test myself
